gocov/convert/convert.go

```diff
@@ -84,29 +114,26 @@ type statement struct {
 	*StmtExtent
 }
 
-func (c *converter) convertProfile(packages packagesCache, p *cover.Profile) error {
-	file, pkgpath, err := findFile(packages, p.FileName)
-	if err != nil {
-		return err
-	}
-	pkg := c.packages[pkgpath]
+func (c *converter) convertPackage(p *cover.Profile, absFilePath, pkgPath string) error {
```

Comment:
I think "convertProfile" is still accurate, and more accurate than "convertPackage". There may be multiple profiles per package, and this method only converts one profile at a time, not a complete package. Could you revert the name please?